common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2656,6 +2667,59 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.SECONDS)
       .createWithDefaultString("10s")
 
+  val WORKER_RATE_LIMIT_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.worker.ratelimit.enabled")
```

Comment:
ratelimit is not that precise, maybe currentControl or congestionControl? WDYT cc @pan3793 @AngersZhuuuu @FMX ?

Reply:
I will use `congestionControl` if others don't complain this.